src/client/ua_client_internal.h

```diff
@@ -168,6 +171,31 @@ struct UA_Client {
     UA_Boolean pendingConnectivityCheck;
 };
 
+static UA_INLINE CustomCallback *CustomCallback_new(void) {
```

Comment:
You can use UA_calloc to get zeroed memory.
So we probably don't need this method.

Reply:
I'll remove this method and replace it with UA_callocs

Reply:
done